guava/pom.xml

```diff
@@ -128,7 +128,7 @@
           <sourcepath>${project.build.sourceDirectory}:${project.build.directory}/jdk-sources</sourcepath>
 
           <!-- Passing `-subpackages com.google.common` breaks things, so we explicitly exclude everything else instead. -->
-          <excludePackageNames>com.google.common.base.internal,com.google.thirdparty.publicsuffix,com.oracle,com.sun,java,javax,jdk,org,sun</excludePackageNames>
+          <excludePackageNames>com.google.common.base.internal.*,com.google.thirdparty.publicsuffix.*,com.oracle.*,com.sun.*,java.*,javax.*,jdk.*,org.*,sun.*</excludePackageNames>
```

Comment:
This PR produces a change in behavior: With this PR applied, the generated Javadoc contains entries for `com.google.common.base.internal`, `com.google.thirdparty.publicsuffix`, and `jdk`. Perhaps we need the `.*` only when that exact package doesn't exist?

Maybe you can find a way that works, but it feels unfortunate to have to further hardcode which packages exist. I guess we may need to cross this bridge eventually, unless we can hope that a future change to the Maven Javadoc Plugin will undo this change?

Reply:
Good catch, it looks like I can use the wildcard `:*` instead of `.*` which will exclude those exact packages, [reference](https://maven.apache.org/plugins/maven-javadoc-plugin/examples/exclude-package-names.html).
I will try test this later.

Reply:
> Good catch, it looks like I can use the wildcard `:*` instead of `.*` which will exclude those exact packages, [reference](https://maven.apache.org/plugins/maven-javadoc-plugin/examples/exclude-package-names.html).
> I will try test this later.

Please ignore this comment, i misread the documentation

Reply:
@cpovirk this change in behavior is fixed in #3478